domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandlinger/stopp/Stoppbehandling.kt

```diff
@@ -0,0 +1,94 @@
+package no.nav.su.se.bakover.domain.behandlinger.stopp
+
+import arrow.core.Either
+import no.nav.su.se.bakover.domain.AktørId
+import no.nav.su.se.bakover.domain.Attestant
+import no.nav.su.se.bakover.domain.Saksbehandler
+import no.nav.su.se.bakover.domain.oppdrag.Utbetaling
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimuleringClient
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimuleringFeilet
+import no.nav.su.se.bakover.domain.oppdrag.utbetaling.UtbetalingPublisher
+import no.nav.su.se.bakover.domain.oppgave.KunneIkkeOppretteOppgave
+import no.nav.su.se.bakover.domain.oppgave.OppgaveClient
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * Behandling for midlertidig stopp av ytelsen(e). Det er mulig å starte den igjen etter dette.
+ * Den første versjonen stopper alle aktive utbetalinger (de som ikke er utbetalt enda).
+ */
+sealed class Stoppbehandling {
+    abstract val id: UUID
+    abstract val opprettet: Instant
+    abstract val sakId: UUID
+    abstract val status: String
+    abstract val utbetaling: Utbetaling
+    abstract val stoppÅrsak: String
+    abstract val saksbehandler: Saksbehandler
+
+    data class Simulert(
+        override val id: UUID,
+        override val opprettet: Instant,
+        override val sakId: UUID,
+        override val utbetaling: Utbetaling,
+        override val stoppÅrsak: String,
+        override val saksbehandler: Saksbehandler
+    ) : Stoppbehandling() {
+
+        companion object {
+            const val STATUS = "SIMULERT"
+        }
+
+        override val status = STATUS
+
+        fun simuler(simuleringClient: SimuleringClient): Either<SimuleringFeilet, Simulert> {
```

Comment:
Kanskje vi ikke har behov for dette i første omgang (ser at det ikke er implementert, men regner med det var tanken på et tidspunkt)? Slik det er nå gjør vi vel en simulering ved opprettelse av behandlingen, og det gir kanskje ikke så mye mening at vi skal kunne simulere de eksakt samme dataene på nytt igjen? Tenker at denne funksjonaliteten først blir relevant når vi f.eks tillater saksbehandler å endre på dato for stans eller lignende.